decidim-core/app/controllers/decidim/errors_controller.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Decidim
+  class ErrorsController < ApplicationController
+    authorize_resource :error_pages, class: false
```

Comment:
Ended up choosing the minimal (but less DRY solution). Also, finally inherited from `ActionController::Base` as you suggested since once you add the necessary functionality, it was actually possible. See c20ad72.